cvmfs/util.cc

```diff
@@ -755,12 +755,13 @@ bool GetUidOf(const std::string &username, uid_t *uid, gid_t *main_gid) {
 bool GetGidOf(const std::string &groupname, gid_t *gid) {
   char buf[16*1024];
   struct group grp;
-  struct group *result;
-  getgrnam_r(groupname.c_str(), &grp, buf, sizeof(buf), &result);
-  if (result == NULL)
-    return false;
-  *gid = result->gr_gid;
-  return true;
+  struct group *result = NULL;
+  if (getgrnam_r(groupname.c_str(), &grp, buf, sizeof(buf), &result) == 0 &&
```

Comment:
Seems to be a false positive of Coverity, afaik.  The documentation of the function claims that `result` will be set to `NULL` if the group isn't found or if there is an error.  Initializing `result` is fine, the rest of the changes make the code unnecessarily more difficult to read, I find.